src/client/index.ts

```diff
@@ -19,26 +103,82 @@ export function removeEventListener<E extends NotificationEvent>(eventType: E['t
     eventEmitter.removeListener(eventType, listener);
 }
 
+/**
+ * Creates a new Notification.
+ *
+ * The Notification will appear in the Notification Center or as a toast if the Center is not visible.
+ *
+ * ```ts
+ * import {create} from 'openfin-notifications';
+ *
+ * create({
+ *      id: "uniqueNotificationId",
```

Comment:
Could maybe do with some text here that makes it clear what happens if a notification is created with a duplicate ID.

Reply:
I have added this in